erpnext/patches/v13_0/set_auto_create_invoice_on_delivery_note.py

```diff
@@ -0,0 +1,9 @@
+import frappe
+
+def execute():
+	frappe.reload_doc("accounts", "doctype", "accounts_settings")
+
+	if frappe.get_meta("Accounts Settings").has_field("auto_create_invoice_on_delivery_note_submit"):
```

Comment:
Delete the old field (remove the Custom Field record) after the checks are done. The migrate is not removing it.